packages/fluentui/react-northstar/src/themes/teams/components/Attachment/attachmentVariables.ts

```diff
@@ -56,7 +56,7 @@ export default (siteVariables: any): AttachmentVariables => ({
   focusColor: undefined,
 
   progressColor: siteVariables.colorScheme.green.background,
-  progressHeight: 4,
+  progressHeight: pxToRem(4),
```

Comment:
nit change as it was really strange to see `pxToRem(v.progressHeight)` in `attachmentStyles.ts`